xml/html_title.xml

```diff
@@ -4111,6 +4111,18 @@
     <param pos="0" name="service.product" value="darkstat"/>
   </fingerprint>
 
+  <fingerprint pattern="^Login - Radarr">
+    <description>Radarr</description>
+    <example>Login - Radarr</example>
+    <param pos="0" name="service.product" value="Radarr"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Login - Sonarr">
```

Comment:
Is there a reason to not have an end of string anchor here? If not, please add the anchor.